### ISSUE TITLE
CI for Python 3.8 + ophyd master build is broken

### ISSUE THREAD

**Issue by danielballan** [bluesky/bluesky#1339]
Per the nightly Travis-CI CRON builds, the  Python 3.8 + ophyd `master` build has been broken for 13 days. Unclear if this is due to a change in ophyd `master` or due to a change in some upstream dependency. One theory is that #1314 is related.

**Comment by tacaswell** [comment]
From the loop warning that #1314 is fixing we have ~30k line of warnings, add to that at least ~50k (and I supect another 40k+ that gets cut off before it hits the bottom) 

```
  /home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/ophyd/utils/__init__.py:163: DeprecationWarning: The signature of a Status callback is now expected to be cb(status). The signature cb() is supported, but support will be removed in a future release of ophyd.

```

The log is being cutoff something around 125k lines.

**Comment by danielballan** [comment]
Closed by #1340 and https://github.com/bluesky/ophyd/pull/852
